### DISCUSSION TITLE
Logfile

### DISCUSSION THREAD

**Discussion by Torinde** [dosbox-staging/dosbox-staging#4032] [Q&A]
I want to keep a log of the console messages, but it seems that is supported only in the debugger executable?
I added to .conf:
```
[log]
logfile     = DOSbox.log.txt
```

`dosbox.exe` gave warning about invalid setting while `dosbox_with_debugger.exe` created an empty log file.

also, unfortunately debugger version console says "stderr verbosity: 0" and then dashes and that's it, nothing else is logged in the console and the log file remains empty (maybe I didn't do drastic enough things to warrant logging while verbosity is zero?).
![image](https://github.com/user-attachments/assets/ed0081cb-6b26-4eb9-a925-a2a87ecbcc88)

So, two questions:
1. Can I enable logfile in normal executable? (it seems not for the moment? #3285)
2. Can I make debugger executable console to log everything like the normal executable? (and hopefully all that to be written in the logfile)

(workaround is to start Staging from a CMD prompt, then the messages remain in the CMD window)

Related:
- #2183

**Comment by johnnovak** [comment]
We don't support writing the logs to a file yet (we should, though). Yeah, that's the workaround to use: start the exe from the command window.

I have never used the debugger-enabled exe, so no idea
